web/libs/ui/src/tokens/colors.scss

```diff
@@ -16,44 +16,44 @@
-  --sand_100: var(--color-primitive-sand-100);
-  --sand_200: var(--color-primitive-sand-200);
-  --sand_300: var(--color-primitive-sand-300);
-  --sand_400: var(--color-primitive-sand-400);
-  --sand_500: var(--color-primitive-sand-500);
-  --sand_600: var(--color-primitive-sand-600);
-  --sand_700: var(--color-primitive-sand-700);
-  --sand_800: var(--color-primitive-sand-800);
-  --sand_900: var(--color-primitive-sand-900);
-  --plum_500: var(--color-primitive-plum-500);
-  --plum_400: var(--color-primitive-plum-400);
-  --plum_200: var(--color-primitive-plum-200);
-  --plum_100: var(--color-primitive-plum-100);
-  --plum_0: var(--color-primitive-plum-000);
-  --persimmon_0: var(--color-primitive-persimmon-000);
-  --persimmon_300: var(--color-primitive-persimmon-300);
-  --persimmon_400: var(--color-primitive-persimmon-400);
-  --canteloupe_0: var(--color-primitive-canteloupe-000);
-  --canteloupe_100: var(--color-primitive-canteloupe-100);
-  --canteloupe_400: var(--color-primitive-canteloupe-400);
-  --canteloupe_500: var(--color-primitive-canteloupe-500);
-  --canteloupe_600: var(--color-primitive-canteloupe-600);
-  --canteloupe_700: var(--color-primitive-canteloupe-700);
-  --kale_0: var(--color-primitive-kale-000);
-  --kale_300: var(--color-primitive-kale-300);
-  --kale_400: var(--color-primitive-kale-400);
+  --sand_0: var(--color-sand-000);
+  --sand_100: var(--color-sand-100);
+  --sand_200: var(--color-sand-200);
+  --sand_300: var(--color-sand-300);
+  --sand_400: var(--color-sand-400);
+  --sand_500: var(--color-sand-500);
+  --sand_600: var(--color-sand-600);
+  --sand_700: var(--color-sand-700);
+  --sand_800: var(--color-sand-800);
+  --sand_900: var(--color-sand-900);
+  --plum_500: var(--color-plum-500);
+  --plum_400: var(--color-plum-400);
+  --plum_200: var(--color-plum-200);
+  --plum_100: var(--color-plum-100);
+  --plum_0: var(--color-plum-000);
+  --persimmon_0: var(--color-persimmon-000);
+  --persimmon_300: var(--color-persimmon-300);
+  --persimmon_400: var(--color-persimmon-400);
+  --canteloupe_0: var(--color-canteloupe-000);
+  --canteloupe_100: var(--color-canteloupe-100);
+  --canteloupe_400: var(--color-canteloupe-400);
+  --canteloupe_500: var(--color-canteloupe-500);
+  --canteloupe_600: var(--color-canteloupe-600);
+  --canteloupe_700: var(--color-canteloupe-700);
+  --kale_0: var(--color-kale-000);
+  --kale_300: var(--color-kale-300);
+  --kale_400: var(--color-kale-400);
   --red_1: #FFF1F0;
   --red_3: #FFCCC7;
   --red_7: #CF1322;
   --red_10: #5C0011;
-  --grape_0: var(--color-primitive-grape-000);
-  --grape_100: var(--color-primitive-grape-100);
+  --grape_0: var(--color-grape-000);
+  --grape_100: var(--color-grape-100);
   --grape_500-raw: 86 111 207;
-  --grape_500: rgb(var(--grape_500-raw));
-  --grape_600: var(--color-primitive-grape-600);
-  --grape_700: var(--color-primitive-grape-700);
-  --grape_800: var(--color-primitive-grape-800);
+  --grape_500: var(--color-grape-500);
```

Comment:
lets remove `-grape_500-raw` if we're no longer using that

Reply:
One thing I have to still figure out, is how we are going to approach the raw values in general. Still sorting that out actually, but also whatever we do there, we have to remember about the other theming that was done against these old raw values.

Reply:
ahh ok, sounds like a follow up PR

Reply:
I added support, but the alignment and usage will definitely come in a follow up PR, likely as part of completing Dark Mode.